contracts/assembly/src/contract.rs

```diff
@@ -674,6 +735,43 @@ pub fn query_proposals(
     })
 }
 
+/// ## Description
+/// Returns proposal voters stored in the vector of [`Addr`].
+/// ## Params
+/// * **deps** is an object of type [`Deps`].
+///
+/// * **proposal_id** is a parameter of type `u64`. This is the proposal identifier.
+///
+/// * **vote_option** is an [`ProposalVoteOption`] type.
+///
+/// * **start_after** is an [`Option`] type. Specifies the proposal list index to start reading from.
+///
+/// * **limit** is a [`Option`] type. Specifies the number of items to read.
+pub fn query_proposal_voters(
+    deps: Deps,
+    proposal_id: u64,
+    vote_option: ProposalVoteOption,
+    start: Option<u64>,
+    limit: Option<u32>,
+) -> StdResult<Vec<Addr>> {
+    let limit = limit.unwrap_or(DEFAULT_LIMIT).min(MAX_LIMIT);
```

Comment:
Querying of 60k voters could be painful with max limit 30. I think MAX_LIMIT can be 200 users or even more.

Reply:
@maksk7 could you find out the real limit?